static/app/types/alerts.tsx

```diff
@@ -50,7 +50,7 @@ export type IssueAlertRuleAction = Omit<
   dynamic_form_fields?: IssueConfigField[];
 } & {
   // These are the same values as the keys in `formFields` for a template
-  [key: string]: number | string;
+  [key: string]: any;
```

Comment:
That's not what I meant but ok

Reply:
they can literally be any primitive type